nextjs-approuter/.env.example

```
@@ -0,0 +1,7 @@
+# This is the Flagsmith environment ID used in the example from YouTube.
+# You can change it to a different ID for testing different flags in your
+# own account.
+#
+# If following the blog tutorial, copy this file to a new file ".env" and update
+# with the server-side key you create from your own account.
+FLAGSMITH_ENVIRONMENT_ID=5zsj2BaedF6BcBHXLNGqUj
```

Careful here, it is a real key
```suggestion
FLAGSMITH_ENVIRONMENT_ID={SERVER_SIDE_FLAGSMITH_KEY}
```

---

Definitely! I only included it because it was originally in this part of the codebase: https://github.com/Flagsmith/flagsmith-js-examples/pull/14/files#diff-80a22e2660bfc952806579ca3945c9b743a2afbc0f6bebfbcc06138e338372c2L23

Normally, I would not include a key in a commit :)

I do like your suggestion though to get rid of the key altogether. Users can experiment with their own key.